README.md

```diff
@@ -76,6 +76,7 @@ config.providers = [
   {
     provider: :rollbar,
     api_key: ENV['ROLLBAR_KEY'],
+    enabled: Rails.env.production?
```

Comment:
I think more people are going to want to turn it off for testing than on for staging, and I wanted to keep this relatively easy to copy. If you think that staging is a normal case to be sending rollbars, I'll add it back in.